### DISCUSSION TITLE
Can VulkanSceneGraph be used to convert .osgb files to the u3d format?

### DISCUSSION THREAD

**Discussion by dwatkins17** [vsg-dev/VulkanSceneGraph#1177] [Q&A]
Before I even get started installing I figured a gut check this is even what I need might be useful. Does VulkanSceneGraph let you convert between 3d formats or should I be looking at openSceneGraph for that?

**Comment by robertosfield** [comment]
Both OpenSceneGraph and VulkanSceneGraph are libraries primarily for rendering not format conversion.  Both can load a range of 3d and image formats rather than writing to other 3rd part formats. A few formats are support where really generally used like image write.

For format conversion you might find something like assimp more helpful.
